src/quoteservice/app/routes.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+use OpenTelemetry\API\Trace\AbstractSpan;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\SDK\Trace\Tracer;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\App;
+
+function calculateQuote($jsonObject, Tracer $tracer): float
+{
+    $quote = 0.0;
+    $childSpan = $tracer
+        ->spanBuilder('calculate-quote')
+        ->setSpanKind(SpanKind::KIND_INTERNAL)
+        ->startSpan();
+    $childSpan->addEvent('Calculating quote');
+
+    try {
+        $numberOfItems = intval($jsonObject['numberOfItems']);
+        $quote = 8.90 * $numberOfItems;
+
+        $childSpan->setAttribute('app.quote.items.count', $numberOfItems);
+        $childSpan->setAttribute('app.quote.cost.total', $quote);
+    } catch (Exception $exception) {
```

Comment:
```suggestion
    } catch (\Exception $exception) {
```
(or, add a `use Exception` above)